JsonUtils/JsonUtils.psm1

```diff
@@ -495,4 +495,4 @@ function Get-SortedPSCustomObjectRecursion
     {
         throw
```

Comment:
Why are you removing throw here?

Reply:
keyboard slip, didn't mean to do that

Reply:
In fact, I'm not sure what's happened here, because the change highlighting is on the `}` of line 498 and it looks the same on both sides. (I used the online editor)

Reply:
Ah, if you hover over it it says the left side doesn't have a newline. The online editor must have added it.